core/src/test/java/io/jenkins/pluginhealth/scoring/probes/SpotBugsProbeTest.java

```diff
@@ -50,16 +50,16 @@
 
 @ExtendWith(MockitoExtension.class)
 class SpotBugsProbeTest extends AbstractProbeTest<SpotBugsProbe> {
-    @Override
-    protected SpotBugsProbe getSpy() {
-        return spy(SpotBugsProbe.class);
-    }
-
     @Test
     public void shouldNotRequireRelease() {
         assertThat(getSpy().requiresRelease()).isFalse();
     }
 
+    @Override
+    protected SpotBugsProbe getSpy() {
+        return spy(SpotBugsProbe.class);
+    }
+
```

Comment:
Again.

Reply:
It is all because of `Ctrl + Alt + L`. I did not do it manually.